lib/discourse_theme/cli.rb

```diff
@@ -167,6 +183,94 @@ def run(args)
         else
           UI.info "Manage: #{client.root}/admin/customize/themes/#{theme_id}"
         end
+      elsif command == "rspec"
+        index = dir.index("/spec")
+        spec_path = "/spec"
+
+        if index
+          spec_path = dir[index..-1]
+          dir = dir[0..index - 1]
+        end
+
+        raise DiscourseTheme::ThemeError.new "'#{dir} does not exist" unless Dir.exist?(dir)
+
+        # Checks if the container is running
+        container_name = DISCOURSE_TEST_DOCKER_CONTAINER_NAME
+        is_running = false
+        container_exists = false
+
+        if !(
+             output =
+               execute(command: "docker ps -a --format '{{json .}}' | grep #{container_name}")
+           ).empty?
+          container_exists = true
+          is_running = JSON.parse(output)["State"] == "running"
+        end
+
+        basename = Pathname.new(dir).basename.to_s
+        verbose = !!args.delete("--verbose")
+        headless = !!args.delete("--headless")
+
+        if !is_running || args.delete("--rebuild")
+          if container_exists
+            execute(command: "docker stop #{container_name}")
+            execute(command: "docker rm -f #{container_name}")
+          end
+
+          execute(
+            command: <<~CMD.squeeze(" "),
+              docker run -d \
+                -p 31337:31337 \
```

Comment:
Hmm this is tricky because the ports have to be exposed before hand and we are not creating a new container each time we need to run the system tests for a theme. For now, I think we should just not support running multiple theme specs at the same time to keep this simple.